pioneer_control/Vec2i32 origin
pioneer_control/Vec2i32 goal
---
pioneer_control/Vec2i32[] path
